patches/marketplace.diff

```diff
@@ -19,22 +19,22 @@ Index: code-server/lib/vscode/src/vs/platform/product/common/product.ts
 ===================================================================
 --- code-server.orig/lib/vscode/src/vs/platform/product/common/product.ts
 +++ code-server/lib/vscode/src/vs/platform/product/common/product.ts
-@@ -45,7 +45,14 @@ else if (typeof require?.__$__nodeRequir
- 	}
+@@ -50,7 +50,14 @@ else if (typeof require?.__$__nodeRequir
+ 		const pkg = require.__$__nodeRequire(joinPath(rootPath, 'package.json').fsPath) as { version: string };
  
- 	Object.assign(product, {
--		version: pkg.version
-+		version: pkg.version,
-+		extensionsGallery: env.EXTENSIONS_GALLERY ? JSON.parse(env.EXTENSIONS_GALLERY) : (product.extensionsGallery || {
-+			serviceUrl: "https://open-vsx.org/vscode/gallery",
-+			itemUrl: "https://open-vsx.org/vscode/item",
-+			resourceUrlTemplate: "https://open-vsx.org/vscode/asset/{publisher}/{name}/{version}/Microsoft.VisualStudio.Code.WebResources/{path}",
-+			controlUrl: "",
-+			recommendationsUrl: "",
-+		}),
- 	});
+ 		Object.assign(product, {
+-			version: pkg.version
++			version: pkg.version,
++			extensionsGallery: env.EXTENSIONS_GALLERY ? JSON.parse(env.EXTENSIONS_GALLERY) : (product.extensionsGallery || {
++				serviceUrl: "https://open-vsx.org/vscode/gallery",
++				itemUrl: "https://open-vsx.org/vscode/item",
++				resourceUrlTemplate: "https://open-vsx.org/vscode/asset/{publisher}/{name}/{version}/Microsoft.VisualStudio.Code.WebResources/{path}",
++				controlUrl: "",
++				recommendationsUrl: "",
++			}),
+ 		});
+ 	}
```

Comment:
Looks like this is behind a `pkg.version` check now so it might not be getting set which means no marketplace.  So probably we just need to move it out of the `if` into its own assignment.

Reply:
Nice catch! Fixed. It changed here fyi: https://github.com/microsoft/vscode/commit/9db57e76e9f16f552203585a394e50cf0f7a84a7#diff-91a3e5c34efae2949de77b7694762982a94f840871a0b76ad44a1c1bdee41ace